src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs

```diff
@@ -108,9 +107,6 @@ private void StartServiceCallback(object sender, EventArgs e)
                     sinkProvider: DefaultSinkProvider
                 );
 
-                var serviceType = typeof(IntegrationService);
-                _marshalledService = RemotingServices.Marshal(_service, serviceType.FullName, serviceType);
```

Comment:
how do we know this line isn't important?

Reply:
I was attempting to try out if is the case or not. I am seeing integration test failures, so possibly related to this removal. I plan to investigate later today, and possibly restore back this code.

Reply:
Reverted, this was causing integration tests to fail on this branch.